packages/data-stores/src/user/actions.ts

```diff
@@ -85,7 +85,7 @@ export function createActions( clientCreds: WpcomClientCredentials ) {
 	};
 }
 
-type ActionCreators = ReturnType< typeof createActions >;
+export type ActionCreators = ReturnType< typeof createActions >;
```

Comment:
Pre-existing, but I found it odd that the types were not declared in the `types` module instead.